src/cogs/redditAPI.py

```diff
@@ -171,21 +202,26 @@ async def sublist(self, ctx):
     @commands.command(name ="addsub", help = addsub_help)
     @commands.has_permissions(administrator = True)
     async def addsub(self, ctx, s):
-        with open('./cogs/subreddit.txt', 'a+') as f5:
-            f5.seek(0)
-            data = f5.read(10000)
-            if len(data) > 0 :
-                f5.write("\n")
-            f5.write(s)
-            f5.close()
-
-        memes_list = []
-        f = open('./cogs/subreddit.txt', 'r')
-        memes_list = f.readlines()
-        f.close()
+
+        subsi_list = []
```

Comment:
Inappropriate naming :
Naming the list as `subsi_list` is quite ambiguous and doesn't clearly indicate the reasoning for such a name. Change all subsequent occurences of the same into some appropriate / self-explanatory name

Reply:
updated ..!!